library/kani_macros/src/lib.rs

```diff
@@ -7,6 +7,7 @@
 // downstream crates to enable these features as well.
 // So we have to enable this on the commandline (see kani-rustc) with:
 //   RUSTFLAGS="-Zcrate-attr=feature(register_tool) -Zcrate-attr=register_tool(kanitool)"
+#![feature(let_chains, proc_macro_diagnostic, box_patterns)]
```

Comment:
😞 okay. They are just so useful. The only one I think we should keep for sure is `proc_macro_diagnostic`, because it allows us to throw clean errors in the macro (e.g. the `thing.span();unwrap().err()` stuff)